dp/cloud/python/magma/test_runner/tests/test_dp_with_orc8r.py

```diff
@@ -245,6 +260,17 @@ def when_cbsds_are_fetched(
         self.assertEqual(len(cbsds), expected_cbsds_num)
         return cbsds
 
+    def when_cbsd_is_fetched(self, serial_number: str = None) -> Dict[str, Any]:
+        params = {}
+        params['serial_number'] = serial_number
```

Comment:
why not just `params = {'serial_number': serial_number}`?

Reply:
Redone as suggested.